server/src/main/java/io/envoyproxy/controlplane/server/DiscoveryServer.java

```diff
@@ -236,7 +270,8 @@ public void onNext(DiscoveryRequest request) {
         requestTypeUrl = defaultTypeUrl;
       }
 
-      LOGGER.info("[{}] request {}[{}] with nonce {} from version {}",
+      LOGGER.info(
```

Comment:
I did follow https://github.com/envoyproxy/java-control-plane/blob/master/CONTRIBUTING.md and am using the google-java-format plugin and the style mentioned. I had to manually revert a lot of automatically converted changes not related to my additions and must have missed this one. The google style guide mentions 100 as the limit and that's what the plugin enforces. I'll update my settings to reflect this, but it would be appreciated if it was mentioned in the docs; or just formatted according to the plugin :-)